ui/app/AppLayouts/Profile/ProfileLayout.qml

```diff
@@ -36,100 +41,198 @@ StatusAppTwoPanelLayout {
         anchors.fill: parent
     }
 
-    rightPanel: StackLayout {
-        id: profileContainer
-
+    rightPanel: Item {
         anchors.fill: parent
 
-        currentIndex: Global.settingsSubsection
-
-        onCurrentIndexChanged: {
-            if(visibleChildren[0] === ensContainer){
-                ensContainer.goToStart();
+        StatusBanner {
+            id: banner
+            anchors.top: parent.top
+            anchors.left: parent.left
+            anchors.right: parent.right
+            visible: {
+                if(profileContainer.currentIndex === Constants.settingsSubsection.wallet &&
+                        profileView.store.walletStore.areTestNetworksEnabled)
+                    return true
+                return false
```

Comment:
nit: could be simplified to: 
```
visible: profileContainer.currentIndex === Constants.settingsSubsection.wallet && profileView.store.walletStore.areTestNetworksEnabled
```

Reply:
@alaibe tbh I just copied that code as it is, anyhow I updated it now as you suggested.